lib/refile/s3.rb

```diff
@@ -60,7 +59,7 @@ def initialize(region:, bucket:, max_size: nil, prefix: nil, hasher: Refile::Ran
     verify_uploadable def upload(uploadable)
       id = @hasher.hash(uploadable)
 
-      if uploadable.is_a?(Refile::File) and uploadable.backend.is_a?(S3) and uploadable.backend.access_key_id == access_key_id
+      if uploadable.is_a?(Refile::File) and uploadable.backend.is_a?(S3) 
```

Comment:
Hey @klausbadelt, thanks for the comments. 

1. Is copying between different AWS users/accounts a currently supported use case?
2. If so, are there unit tests we can add to catch problems?